src/framework/mocha/Utilities/MOFunctionArgument.m

```diff
@@ -1095,4 +1095,55 @@ + (NSInteger)structureToJSValue:(JSValueRef *)value inContext:(JSContextRef)ctx
     return c - c0 - 1;
 }
 
++ (BOOL)isHighSierraOrHigher {
```

Comment:
I'm assuming the CocoaScript stuff has no dependancy on Chocolat?

There are methods like this in `NSObject+Chocolat.m`; `BCRunningSierra` etc. I think it would nicer to add a `BCRunningHighSierra` method there and use that instead. But I'm guessing we can't do that as it would introduce a dependancy we don't want?